### ISSUE TITLE
Introduce Advanced Auton and Ramsete

### ISSUE THREAD

**Issue by victorhu3** [Team41Robotics/RoboGuide#29]
This is a more advanced topic -- so not very high on the priority list. We never documented this year's auton code very well so we might as well do some more here. I know the guy on Hillsborough literally wrote his own pure pursuit, so we might want to step up our auton game and implant these ideas in their heads earlier.

**Comment by peterapps** [comment]
Good idea. I think that's something that could definitely be its own subheading on the Robot Code page, especially because it wasn't always intuitive to implement.

If you're not familiar with the web dev stuff and want to just type up a Google Doc or markdown file that's fine. Then someone else can copy it over and format it in HTML and React. Same applies for any other documentation you're interested in contributing.
